jaxrs/src/main/java/org/jboss/aerogear/unifiedpush/rest/sender/PushNotificationSenderEndpoint.java

```diff
@@ -103,9 +110,11 @@ public Response send(final InternalUnifiedPushMessage message, @Context HttpServ
 
         // add the client identifier
         message.setClientIdentifier(HttpRequestUtil.extractAeroGearSenderInformation(request));
-
-        // submitted to EJB:
-        notificationRouter.submit(pushApplication, message);
+        
+        // start the producer and push a message to "agpush_pushMessageProcessing" topic
+        pushMessageProcessingProducer.send(KAFKA_PUSH_MESSAGE_PROCESSING_TOPIC, pushApplication, message);
+        
+        
         logger.debug(String.format("Push Message Request from [%s] API was internally submitted for further processing", message.getClientIdentifier()));
```

Comment:
We use slf4j: using String.format in log messages when using slf4j is discouraged because the string interpolation is performed even if the log is disabled. Please, rewrite using the slf4j syntax.